<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressSetDialog</class>
 <widget class="QDialog" name="AddressSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frameAddressSetting">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_address_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>73</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px</string>
   </property>
   <property name="text">
    <string>本地IP地址</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMacAddr">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>283</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font:24px</string>
   </property>
   <property name="text">
    <string>00:01:02:03:04:05</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditManageCenter1Name">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>423</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font:24px</string>
   </property>
   <property name="text">
    <string>9999-0001</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditNativeName">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>353</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font:24px</string>
   </property>
   <property name="text">
    <string>1111-0001</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_address_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>143</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px</string>
   </property>
   <property name="text">
    <string>子网掩码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_address_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>353</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px</string>
   </property>
   <property name="text">
    <string>本机名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditGateway">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>213</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font:24px</string>
   </property>
   <property name="text">
    <string>192.168.1.1</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_address_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>423</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px</string>
   </property>
   <property name="text">
    <string>管理中心机名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_address_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>213</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px</string>
   </property>
   <property name="text">
    <string>网       关</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_address_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>493</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px</string>
   </property>
   <property name="text">
    <string>中心计算机名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_address_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>283</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px</string>
   </property>
   <property name="text">
    <string>MAC地址</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditNativeIP">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>73</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font:24px</string>
   </property>
   <property name="text">
    <string>192.168.1.132</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditNetmask">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>143</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font:24px</string>
   </property>
   <property name="text">
    <string>255.255.255.0</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditManageCenter2Name">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>493</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font:24px</string>
   </property>
   <property name="text">
    <string>9999-0002</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>10</y>
     <width>510</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>网络</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
